roseau/load_flow/models/lines/parameters.py

```diff
@@ -677,50 +708,27 @@ def _from_geometry(
             material = _DEFAULT_MATERIAL[line_type]
         if insulator is None:
             insulator = _DEFAULT_INSULATOR[line_type]
+        if material_neutral is None:
+            material_neutral = material
+        if insulator_neutral is None:
+            insulator_neutral = insulator
```

Comment:
So here is the problem with this. `insulator_neutral = None` is confusing because we cannot know if it means "no insulation" or it means "no value provided, please use the value of the phases insulator". In other words, the user cannot create a non-insulated neutral with insulated phases.

I suggest we keep the `NONE` enumeration on the `Insulator` class to avoid this confusion here or elsewhere.